Add note on sending null values (#5441)
segmentio/segment-docs: src/connections/destinations/catalog/salesforce-marketing-cloud/index.md

This issue usually occurs for customers that have very large volumes of customer data (10MM+ users), because multiple audiences and traits attempt to send large quantities of backfill data into SFMC at the same time, and compete for the SFMC rate limit. To help with this, avoid syncing multiple *new* audiences and *new* traits at the same time. Instead, create an audience, sync it to SFMC and wait for it to complete. Then, create and sync your next audience or trait.

You can also request a higher rate limit from your SFMC account representative. After you confirm the higher rate limit with your SFMC representative, contact [Segment Product Support](http://segment.com/help/contact) to adjust the rate limit from the Segment side for you.

### Sending null values

> Events containing a field with a `null` value will be silently rejected by SFMC's REST API. To prevent this, especially in situations where events require transformation prior to sending to SFMC, consider employing an [Insert Function](/docs/connections/functions/insert-functions/) to replace any `null` values within your events.
> 
> If you are connecting to this destination using [Reverse ETL](/docs/connections/reverse-etl/), please ensure that the model connected to the mapping is configured to return non-null values to avoid any disruptions.

Reviewer: ```suggestion
Events containing a field with a `null` value are silently rejected by SFMC's REST API. To prevent this, especially in situations where events require transformation prior to sending to SFMC, consider employing an [Insert Function](/docs/connections/functions/insert-functions/) to replace any `null` values within your events.

If you're connected to this destination using [Reverse ETL](/docs/connections/reverse-etl/), ensure that you configure the model connected to the mapping to return non-null values to avoid any disruptions.
```
